{"name": "Test - create record twice ", "status": "failed", "statusDetails": {"message": "AssertionError: assert 504 == 200\n +  where 504 = <Response [504]>.status_code", "trace": "self = <testcase.ACS.test_workflow.TestWorkflow object at 0x0000021CA09A8370>\nbase_api = <utils.base_api_acs.BaseAPIACS object at 0x0000021CA35BF010>\n\n>   ???\nE   assert 504 == 200\nE    +  where 504 = <Response [504]>.status_code\n\nC:\\actions-runner\\actions-runner-win-x64-2.324.0\\_work\\api_automation\\testcase\\ACS\\test_workflow.py:38: AssertionError"}, "attachments": [{"name": "Request Info", "source": "23bd2365-6520-4f59-b13e-5e8a0277dc48-attachment.json", "type": "application/json"}, {"name": "Response Info", "source": "e4d40a40-5032-4a2e-b814-f5abb553dadb-attachment.json", "type": "application/json"}], "parameters": [{"name": "base_api", "value": "'dev'"}], "start": 1748327999816, "stop": 1748328030786, "uuid": "cccdae84-35a1-45fb-89c1-7baac3c57298", "historyId": "eb9caf1256f6b8b5f2af2999553fd09d", "testCaseId": "7086946e47212c59436c76a97ea6790f", "fullName": "testcase.ACS.test_workflow.TestWorkflow#test_create", "labels": [{"name": "parentSuite", "value": "testcase.ACS"}, {"name": "suite", "value": "test_workflow"}, {"name": "subSuite", "value": "TestWorkflow"}, {"name": "host", "value": "Cherry-640879"}, {"name": "thread", "value": "51132-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.ACS.test_workflow"}]}